### PR TITLE
remove hardcoded height

### LINKED ISSUE #18756: [Bug]: Text Overflow in Asset section while transferring tokens
### Describe the bug

https://user-images.githubusercontent.com/93417120/233865736-58754e51-c7d9-4276-8792-8ce6103598ca.mov

### Steps to reproduce

1. Select an account with non-zero balance
2. Click Send
3. Add a public wallet address
4. Observe asset section

### Error messages or log output

_No response_

### Version

10.28.3

### Build type

None

### Browser

Brave

### Operating system

MacOS

### Hardware wallet

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by thebinij**
## Explanation

* Fixes #18756 

### Before

https://github.com/MetaMask/metamask-extension/assets/93417120/04a17edc-8c26-459a-b62f-4017ebebb638

### After

https://github.com/MetaMask/metamask-extension/assets/93417120/743b7052-25a9-4075-a045-c4d1c879430b

## Manual Testing Steps
- Select an account whose balance has large floating point number(ex: x.xxxxxxxx)
- Click 'Send'
- Add Wallet Address
- Look at Balance within the 'Asset' section

## Pre-merge author checklist

- [X] I've clearly explained:
  - [X] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [X] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/css/itcss/components/send.scss [hunk 1]
```diff
@@ -585,7 +585,6 @@
   }
 
   &__asset-dropdown {
-    height: 54px;
```
Comment (digiwand) [inline ui/css/itcss/components/send.scss:588]: I'm a fan of this fix. Thanks for proposing this!

One suggestion, I think making this `min-height: 54px` would ensure the height will also not shrink.

Comment (thebinij) [reply ui/css/itcss/components/send.scss:588]: Thanks for the suggestion. 👍

**Review by darkwing — APPROVED** [review]

**Comment by thebinij** [comment]
Hi @digiwand, would you please review this PR ? Thanks

**Review by digiwand — APPROVED** [review]
awesome, thanks for the tag! LGTM!

**Comment by thebinij** [comment]
Hi @digiwand, @darkwing. I don't have write access to merge this repo. Could I get the write access? If no, would you please merge this PR. Thanks.

**Review by digiwand — APPROVED** [review]

**Pull request closed without merging**